spring-cloud-stream-core-docs/src/main/asciidoc/spring-cloud-stream-overview.adoc

```diff
@@ -1539,7 +1539,7 @@ For Spring Cloud Stream samples, please refer to the https://github.com/spring-c
 == Getting Started
 
 To get started with creating Spring Cloud Stream applications, visit the https://start.spring.io[Spring Initializr] and create a new Maven project named "GreetingSource".
-Select Spring Boot version 1.3.4 SNAPSHOT and search or tick the checkbox for Stream Kafka (we will be using Kafka for messaging).
+Select the Spring Boot version that works for you and search or tick the checkbox for Stream Kafka (we will be using Kafka for messaging).
```

Comment:
yeah, I will give a try to generate this property value from the current `spring-boot` version that the project uses and say something like `:spring-boot-supported-versions: starting from 1.3.x to <currently-used-version>`